uber/custom_tags.py

```diff
@@ -17,23 +17,31 @@ def safe_string(text):
 
 @JinjaEnv.jinja_filter(name='datetime')
 def datetime_filter(dt, fmt='%-I:%M%p %Z on %A, %b %e'):
-    return ' '.join(dt.strftime(fmt).split()).replace('AM', 'am').replace('PM', 'pm') if dt else ''
+    if not dt:
+        return ''
```

Comment:
Hm, yeah, in cases like these I guess what I tend to do is keep it on one line, but 'promote' the shorter return to be first. So I probably would have rewritten it as `return '' if not dt else ' '.join(dt.strftime(fmt).split()).replace('AM', 'am').replace('PM', 'pm')`. This is just as good and arguably more readable though.